community/contributing/pr_workflow.rst

```diff
@@ -418,9 +418,9 @@ will raise an error:
     $ git push origin better-project-manager
     To https://github.com/akien-mga/godot
      ! [rejected]        better-project-manager -> better-project-manager (non-fast-forward)
-    error: failed to push some refs to 'https://akien-mga@github.com/akien-mga/godot'
+    error: Failed to push some refs to 'https://akien-mga@github.com/akien-mga/godot'
```

Comment:
No, we're not changing the case in copy-pasted error messages from other programs...